Cargo.toml

```diff
@@ -2,10 +2,12 @@
 members = [
   "audio",
   "backends/gstreamer",
+  "backends/gstreamer/source-element",
+  "backends/gstreamer/source-element-plugin",
```

Comment:
As a gstreamer plugin can hold a set of different elements, and we could end up adding more custom elements for servo, I wouldn't call source-element-plugin, but servo-plugin or so.

Reply:
Also you don't really need a plugin at all, it only complicates things if this is only supposed to be used from servo itself and not external applications. All you need to do is to call `gst::Element::register()` at some point, no need for all the plugin stuff.

Reply:
Ah, you do that in https://github.com/servo/media/pull/179/commits/eaf2cdf864867af9679de57ef315eeff1cb5ea62

Reply:
Yes, this commit oriented review doesn't work... I should have reviewed the whole diff :D

Reply:
Maybe the commits should be squashed together to allow each of the commits to be meaningful :)

Reply:
Yeah, sorry about that :$